src/action/autopilot.js

```diff
@@ -48,6 +54,62 @@ class AtplAction {
       this._notification.display({ msg: 'Error toggling autopilot', err });
     }
   }
+
+  /**
+   * Update node scores to get better channels via autopilot.
+   * @return {Promise<undefined>}
+   */
+  async updateNodeScores() {
+    try {
+      await this._setNetwork();
+      const scores = await this._readNodeScores();
+      await this._setNodeScores(scores);
+    } catch (err) {
+      log.error('Updating autopilot scores failed', err);
+    }
+  }
```

Comment:
I split the high level api into smaller functions. This makes the code much more readable than putting everything in a single try/catch block with nested if/else statements. We should try to split actions up like this for better readability.